apps/financial-aid/web-osk/src/components/UserProvider/UserProvider.tsx

```diff
@@ -22,11 +25,14 @@ interface Props {
 export const UserContext = createContext<UserProvider>({})
 
 const UserProvider = ({ children }: Props) => {
+  const router = useRouter()
```

Comment:
Not this PR, but can we change the probs object name to children or destructure it on line 27?